redis/src/test/scala/zio/redis/KeysSpec.scala

```diff
@@ -1,20 +1,28 @@
 package zio.redis
 
+import java.time.{ Duration => JavaDuration }
+
 import zio.clock.Clock
 import zio.duration._
+import zio.redis.RedisError.ProtocolError
 import zio.test.Assertion._
 import zio.test.TestAspect._
 import zio.test._
 import zio.test.environment.{ TestClock, TestConsole, TestRandom, TestSystem }
 import zio.{ Chunk, Has, ZIO }
 
+object KeysSpec {
+  val migrateTimeout: Duration = Duration.fromMillis(5000)
```

Comment:
Let's use zio's duration please. It delegates to java duration anyhow, but the syntax is a bit nicer.